index.html

```diff
@@ -107,13 +110,19 @@ <h2 id="docconformance">
         used. Where a cell in the third column includes the term
         <dfn><strong>Any</strong> `role`</dfn> it indicates that any `role`
         value apart from the <a>implicit ARIA semantics</a> `role` value, MAY
-        be used. If an cell in the third column includes the term <dfn><strong>No `role`</strong></dfn> it indicates that authors MUST NOT overwrite the implicit ARIA semantics, or native semantics of the HTML element.
+        be used. If a cell in the third column includes the term 
+        <dfn><strong>No `role`</strong></dfn> it indicates that authors MUST NOT
+        overwrite the implicit ARIA semantics, or native semantics of the HTML element.
       </p>
 
       <p class="note" id="aria-usage-note">
         Setting an ARIA `role` and/or `aria-*` attribute that matches the
-        <span>implicit ARIA semantics</span> is unnecessary and is NOT
-        RECOMMENDED as these properties are already set by the browser.
+        <span>implicit ARIA semantics</span> is NOT RECOMMENDED. It is unnecessary,
+        as these properties are already set by the browser. However, explicitly
+        setting these attributes won't generally have an adverse effect (provided
+        that they are correct and indeed match the implicit semantics) - and this
```

Comment:
Prefer a comma here?

```suggestion
        that they are correct and indeed match the implicit semantics), and this
```